modules/apps/frontend-js/frontend-js-spa-web/src/main/java/com/liferay/frontend/js/spa/web/internal/servlet/taglib/SPATopHeadJSPDynamicInclude.java

```diff
@@ -50,7 +50,21 @@ public void include(
 			HttpServletResponse httpServletResponse, String key)
 		throws IOException {
 
-		SPAHelper spaHelper = _spaHelperSnapshot.get();
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		Group group = themeDisplay.getScopeGroup();
+
+		long siteGroupId = group.getGroupId();
```

Comment:
Did this pass SF? :thinking:  I think it should complain about creating a variable for just one use (ie: it would prompt you to pass `group.getGroupId()` to `SPAHelper` directly).